lib/starport_app.dart

```diff
@@ -19,7 +18,6 @@ class StarportApp extends StatelessWidget {
   static late TransactionsStore transactionsStore;
   static late LiquidityStore liquidityStore;
   static late BaseEnv baseEnv;
-  static late NetworkInfo networkInfo;
```

Comment:
getting rid of networkInfo from here, since its now part of `BaseEnv`

Reply:
Noted